packages/client/src/lib.js

```diff
@@ -203,10 +209,10 @@ class Web3Storage {
       const res = await pRetry(
         async () => {
           await rateLimiter()
-          const request = await fetch(url.toString(), {
+          const request = await (customFetch||fetch)(url.toString(), {
```

Comment:
If we default fetch in the constructor, this line is neater
```suggestion
          const request = await fetch(url.toString(), {
```